pkg/booleanpolicy/networkpolicy/networkpolicy.go

```diff
@@ -0,0 +1,31 @@
+package networkpolicy
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/booleanpolicy/augmentedobjs"
+)
+
+// GetNetworkPoliciesApplied creates an augmentedobj.NetworkPoliciesApplied object
+// based on a provided map of network policies.
+func GetNetworkPoliciesApplied(networkPolicies map[string]*storage.NetworkPolicy) *augmentedobjs.NetworkPoliciesApplied {
```

Comment:
What is the map key?

Reply:
The map stores mapping of: policy-id -> policy object